extensions/wikibase/src/org/openrefine/wikibase/schema/WbVariableExpr.java

```diff
@@ -90,12 +92,20 @@ public void setColumnName(String columnName) {
      * Evaluates the expression in a given context, returning
      * 
      * @throws QAWarningException
+     * @throws SpecialValueNoValueException
+     * @throws SpecialValueSomeValueException
      */
     @Override
     public T evaluate(ExpressionContext ctxt)
-            throws SkipSchemaExpressionException, QAWarningException {
+            throws SkipSchemaExpressionException, QAWarningException, SpecialValueNoValueException, SpecialValueSomeValueException {
         Cell cell = ctxt.getCellByName(columnName);
         if (cell != null) {
+            if ("#NOVALUE#".equals(cell.toString())) {
+                throw new SpecialValueNoValueException();
+            } else if ("#SOMEVALUE#".equals(cell.toString())) {
+                throw new SpecialValueSomeValueException();
```

Comment:
Since the `"#NOVALUE#"` and `"#SOMEVALUE#"` Strings have a special meaning defined by this class, I still think it would be good to have them as `public static final String` in this class, to make them part of the public interface of this class. This could be reused in tests, initially.